src/components/authors/authorList.js

```diff
@@ -22,7 +23,7 @@ var AuthorList = React.createClass({
 			return (
 				<tr key={author.id}>
 					<td><a href="#" onClick={this.deleteAuthor.bind(this, author.id)}>Delete</a></td>
-					<td><Link to="manageAuthor" params={{id: author.id}}>{author.id}</Link></td>
+					<td><Link to={"author/" + author.id}>{author.id}</Link></td>

```

Comment:
With React Router 4, you specify the actual path. The named route feature was removed.